.github/CONTRIBUTORS-template.md

```diff
@@ -24,7 +24,7 @@ accept and merge pull requests.
 
 For Django Dash 2010, @pydanny and @audreyr created Django Packages.
 
-They are joined by a host of core developers and contributors. See https://docs.djangopackages.org/en/latest/contributors.html
+They are joined by a host of core developers and contributors. See https://docs.djangopackages.org/en/latest/contributors
```

Comment:
I don't think we have this page in the docs. Should we refer to https://github.com/djangopackages/djangopackages/blob/main/CONTRIBUTORS.md?

Reply:
Ah, you are right. I wasn't paying too much attention and thought this was a link to the contribution page (https://docs.djangopackages.org/en/latest/contributing).